data/mods/Aftershock/scenarios.json

```diff
@@ -90,8 +92,83 @@
     "points": 1,
     "description": "What was to be a routine cargo transfer ended in tragedy when, in a brief moment of chaos, your space ship was intercepted and destroyed by a StO missile.  As soon as the MAW alarm flared to life, you scrambled to the nearest escape pod and barely managed to reach the uncertain safety of the planet below.",
     "allowed_locs": [ "sloc_escape_pod" ],
-    "professions": [ "afs_espatier", "afs_rating" ],
+    "professions": [ "afs_espatier", "afs_rating", "afs_captain_bearman" ],
```

Comment:
I'd keep the profession out of this scenario. It seems a bit to strong for it, due to the eventuall starting with environmental gear.